composer.json

```diff
@@ -38,17 +38,6 @@
     {
       "type": "vcs",
       "url": "https://github.com/christophermancini/php-protobuf"
-    },
-    {
-      "type": "package",
-      "package": {
-        "name": "basho/riak_pb",
-        "version": "2.1.3.0",
-        "dist": {
-          "url": "https://github.com/basho/riak_pb/archive/2.1.3.0.zip",
-          "type": "zip"
-        }
-      }
```

Comment:
PS check out #8, which I would like to get merged before this branch. I wanted to isolate the protobuf generation changes from the Hll changes.